packages/kitchen-sink/src/examples/quiz.tsx

```diff
@@ -0,0 +1,136 @@
+import  { useState } from 'react';
+import { block } from 'million/react';
+import styled from 'styled-components';
+
+const QuizContainer = styled.div`
+  text-align: center;
+  max-width: 600px;
+  margin: 0 auto;
+  padding: 20px;
+  background-color: #f5f5f5;
+  border-radius: 10px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+`;
+
+const QuestionContainer = styled.div`
+  padding: 20px;
+  background-color: #fff;
+  border-radius: 10px;
+  margin-bottom: 20px;
+`;
+
+const QuestionText = styled.h2`
+  margin-bottom: 10px;
+`;
+
+const OptionsContainer = styled.form`
+  display: flex;
+  flex-direction: column;
+`;
+
+const OptionLabel = styled.label`
+  display: block;
+  margin-bottom: 10px;
+`;
+
+const OptionInput = styled.input`
+  margin-right: 5px;
+`;
+
+const NextButton = styled.button`
+  background-color: #007bff;
+  color: #fff;
+  border: none;
+  padding: 10px 20px;
+  border-radius: 5px;
+  cursor: pointer;
+  transition: background-color 0.3s;
+
+  &:hover {
+    background-color: #0056b3;
+  }
+`;
+
```

Comment:
Can we move the styling to a different file please? It helps with the general readabilty.